packages/rules-engine/src/lib/custom-rule-engine/helpers/cast-compare-value.ts

```diff
@@ -0,0 +1,10 @@
+import { CustomRuleInput } from '@/lib/custom-rule-engine/types/custom-rule-input';
+import { castValue } from '@/lib/custom-rule-engine/helpers/cast-value';
+
+export const castCompareWithEval = (options: CustomRuleInput['options'], value: string) => {
+  if (value.startsWith('`') && value.endsWith('`')) {
```

Comment:
I would prefer a different indicator for a JS code. `options.isJavascript` or something.

Reply:
any suggestions?
I am moving the validation to a method e.g.
`
const isJavascript = (value: string) => {
  return value.startsWith('`') && value.endsWith('`');
};
`

is that good enough?

Reply:
No IMO.
But it's up to you to decide.